schema/layer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "/layer",
+  "type": "object",
+  "properties": {
+    "layer": { "type": "string" },
+    "count": { "type": "number" },
+    "geometry": { "type": "string" },
```

Comment:
I think so. There's are the only types I see come up in existing stats. I guess the current code does not exclude the possibility of other types somehow cropping up. Pretty sure Studio is expecting it to be one of these three, though.